From 9182fb378614abb2ff0245fe71a753f22eaaa906 Mon Sep 17 00:00:00 2001
From: Debarshi Ray <debarshir@gnome.org>
Date: Tue, 11 Jun 2013 14:58:21 +0000
Subject: google: Bump credentials generation

Access to the following were turned on in the Google APIs Console:
 - Calendar API
 - Google Calendar CalDAV API
 - Google Contacts CardDAV API

Of these, only the last two are new. We were already requesting the
scope for Calendar API, but looks like the APIs Console is the way
to go now. Interestingly the APIs Console does not list all the other
services that we are interested in, or it is does but is not obvious
to me.

In any case we need access to their new CalDAV API which works with
OAuth2 because that would let us work with 2-factor authenticated
accounts again.

See: https://bugzilla.gnome.org/show_bug.cgi?id=686804
     https://bugzilla.gnome.org/show_bug.cgi?id=688364
---
(limited to 'src/goabackend/goagoogleprovider.c')

diff --git a/src/goabackend/goagoogleprovider.c b/src/goabackend/goagoogleprovider.c
index 8271711..79f5f73 100644
--- a/src/goabackend/goagoogleprovider.c
+++ b/src/goabackend/goagoogleprovider.c
@@ -148,7 +148,7 @@ get_scope (GoaOAuth2Provider *provider)
 static guint
 get_credentials_generation (GoaProvider *provider)
 {
-  return 3;
+  return 4;
 }
 
 static const gchar *

